include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -227,10 +227,8 @@ compared to 1*64 when the hasLeadingOffset is false.
                      "unsigned":$typeWidthInBit,
                      "bool":$needTrans), [{
 
-        // ---- begin GFX908/GFX90A ----
-        auto mfmaEnc = dotOpEnc.getParent().dyn_cast<AMDMfmaEncodingAttr>();
-
-        if (mfmaEnc) {
+        // ---- begin GFX908/GFX90A/GFX11 ----
+        if (dotOpEnc.getParent().isa<AMDMfmaEncodingAttr, AMDWmmaEncodingAttr>()) {
```

Comment:
I think it is better to split this in two independent parts.

MFMA has tons of variants, which adds complexity not needed for WMMA.